src/shared/services/case-fields/format-translator.service.spec.ts

```diff
@@ -30,5 +30,48 @@ describe('FormatTranslatorService', () => {
       // tslint:disable-next-line:quotemark
       expect(formatTranslator.translate("dd MMM ''yyyy''")).toBe('dd MMM yyyy')
     })
+
+    it ('should remove time', () => {
+      expect(formatTranslator.removeTime('YYYY/MM/DD hh:mm:ss')).toBe('YYYY/MM/DD ::');
```

Comment:
This output is a bit unpleasant... shouldn't it simply be `YYYY/MM/DD` without the extraneous ` ::`? 😬

Reply:
I could put something in to remove colons when removing time from a datetime however there is no guarantee that colons will be used to separate the time. Could be 'YYYY:MM:DD hh-mm-ss' in some scenarios. I can make a reasonable assumption that this might be the case however - could just remove all colons if time removed?

Reply:
I suppose you _could_ end up with a really stupid format like `'YYYY:hh-DD.mm MMM'` and then you'd be pretty screwed, although that's unlikely. There _is_ a way to sort this out once and for all (via regex) but it's probably more work than is justified. I think making some assumptions about what constitutes a sensible format is probably reasonable - i.e., `/`, `-`, ` `, and maybe `.` are reasonable separators for the date... so strip out anything else... and then do a trim on the whole string to get rid of any weird whitespace at the edges...?

Reply:
I'd rather not strip everything else out - I think that's a bit of an exclusive assumption and means any additional characters that we're not considering that the user might want are stripped out. However, definitely removing colons and trimming is valid, leaving an explanation in the comments stating why

Reply:
Change made